symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkDatafeedConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.symphony.bdk.core.config.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.io.File;
+
+@Getter
+@Setter
+public class BdkDatafeedConfig {
+
+    private String version = "v1";
+    private String datafeedIdFilePath;
```

Comment:
as we are in the datafeed path of the configuration should we add datafeed again?
config:
  datafeed:
   idFilePath: ?